web/docs/docusaurus.config.js

```diff
@@ -1,93 +1,93 @@
 module.exports = {
```

Comment:
?

Reply:
Yeah, I'll go back through and remove formatting changes